plans/install-upgrade.fmf

```diff
@@ -12,7 +12,7 @@ prepare:
       - python3-devel
       - unzip
   - how: shell
-    script: ansible-galaxy collection install community.general ansible.posix
+    script: ansible-galaxy collection install https://ansible-collection.s3.amazonaws.com/ansible-posix-1.5.4.tar.gz https://ansible-collection.s3.amazonaws.com/community-general-8.5.0.tar.gz
```

Comment:
OK, but it seems better to retry this task in a loop; this approach seems to be bypassing their metadata servers; e.g. what if they decided to store things in GCP or quay.io instead?

Reply:
I think saving to quay.io adds complex. Both GCP and S3 work for me.
BTW, I've tried to run ansible inside a container, but I dropped that for two reasons:
1. it adds a lot of complexity and test code is not easy to understand.
2. libvirt scenario blocked me.